build.sbt

```diff
@@ -63,8 +63,8 @@ lazy val root = (project in file("."))
       ("org.scalanlp" %% "breeze" % "2.1.0"),
       ("org.scalanlp" %% "breeze-natives" % "2.1.0"),
       "ch.unibas.cs.gravis" % "scalismo-niftijiojar" % "0.1.0",
-      "com.lihaoyi" %% "upickle" % "2.0.0",
-      "ch.unibas.cs.gravis" % "hdf5javanatives" % "0.1.0",
+      //"com.lihaoyi" %% "upickle" % "2.0.0",
+      ("ch.unibas.cs.gravis" %% "scalismo-hdf5-json" % "0.1.0-SNAPSHOT").cross(CrossVersion.for2_13Use3),
```

Comment:
I would be in favor of having here a non SNAPSHOT version, or is there a specific reason for it?

Reply:
This will be a non-snapshot version in the final version. But I think we should plan a phase for testing the new io classes with real use cases before the release. As scalismo-hdf5-json is brand new, I expect quite that we will encounter some issues. 

I propose the following strategy:

- Test everything together with Scalismo-ui using the SNAPSHOT release
- Once we have initial tests done, create RCs for all libraries and test those
- Once the RCs are tested for some weeks, we release the final versions.

Reply:
That makes sense.